### PR TITLE
Update ADA rosinstalls.

### PR DESCRIPTION
**Pull request by egordon**
Switch out old ADA rosinstalls for one that works with current feeding demo.

### CODE REVIEW

**Comment by brianhou** [comment]
I suggested deleting all the old rosinstalls and only keeping the new ones that reflect our current stack. I think it would be a good idea to have all the ones that you suggest, but those could perhaps be resolved in future PRs?

**Comment by egordon** [comment]
@gilwoolee @brianhou I just added an ada-feeding-sim rosinstall, since that was easy to make, but I agree that it would be good to unblock this PR and look to future PRs to add other, more minimal rosinstalls. I added place-holder issue #54

**Comment by brianhou** [comment]
Is there some order that the repositories are listed in? If not, it'd be helpful to do something like alphabetical so it's easier to pick out differences.

File: ada-feeding-sim.rosinstall [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+- git: {local-name: ada_description, uri: 'https://github.com/personalrobotics/ada_description.git'}
+- git: {local-name: ada_launch, uri: 'https://github.com/personalrobotics/ada_launch.git'}
+- git: {local-name: aikido, uri: 'https://github.com/personalrobotics/aikido.git'}
+- git: {local-name: libada, uri: 'https://github.com/personalrobotics/libada.git'}
+- git: {local-name: pose_estimators, uri: 'https://github.com/personalrobotics/pose_estimators.git'}
+- git: {local-name: pr_assets, uri: 'https://github.com/personalrobotics/pr_assets.git'}
+- git: {local-name: pr_control_msgs, uri: 'https://github.com/personalrobotics/pr_control_msgs.git'}
+- git: {local-name: pr_ros_controllers, uri: 'https://github.com/personalrobotics/pr_ros_controllers.git'}
+- git: {local-name: pr_tsr, uri: 'https://github.com/personalrobotics/pr_tsr.git'}
+- git: {local-name: rewd_controllers, uri: 'https://github.com/personalrobotics/rewd_controllers.git'}
```
Comment (gilwoolee) [inline ada-feeding-sim.rosinstall]: Need to add the following repositories:
`bite_selection_package`
`food_detector`
`pytorch_retinanet`
(These are no longer submodules of food_detector)
I also want to add `sim_face_detector` to `face_detection` repository and make that a sim-dependency as well.
I can work on that tomorrow morning and let you know when it's ready. 

Why do we need rewd_controllers, pr_ros_controllers, and pr_control_msgs here?

File: ada-feeding.rosinstall [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- git: {local-name: jaco_hardware, uri: 'https://github.com/personalrobotics/jaco_hardware.git'}
+- git: {local-name: libada, uri: 'https://github.com/personalrobotics/libada.git'}
+- git: {local-name: net-ft-ros, uri: 'https://github.com/personalrobotics/net-ft-ros.git'}
+- git: {local-name: pose_estimators, uri: 'https://github.com/personalrobotics/pose_estimators.git'}
+- git: {local-name: pr_assets, uri: 'https://github.com/personalrobotics/pr_assets.git'}
+- git: {local-name: pr_control_msgs, uri: 'https://github.com/personalrobotics/pr_control_msgs.git'}
+- git: {local-name: pr_hardware_interfaces, uri: 'https://github.com/personalrobotics/pr_hardware_interfaces.git'}
+- git: {local-name: pr_ros_controllers, uri: 'https://github.com/personalrobotics/pr_ros_controllers.git'}
+- git: {local-name: pr_tsr, uri: 'https://github.com/personalrobotics/pr_tsr.git'}
+- git: {local-name: rewd_controllers, uri: 'https://github.com/personalrobotics/rewd_controllers.git'}
```
Comment (gilwoolee) [inline ada-feeding.rosinstall]: As mentioned above, we need `bite_selection_package` and `pytorch_retinanet` as well.

**Comment by gilwoolee** [comment]
Looks mostly good except for the requested changes, but I'd like to wait until we test this on a clean workspace, for both sim and real.

**Pull request merged**